Ports/SuperTuxKart/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SuperTuxKart
+useconfigure="false"
+version="1.4"
+archive_hash="9890392419baf4715313f14d5ad60746f276eed36eb580636caf44e2532c0f03"
```

Comment:
nit: please use single quotes for values that don't do interpolation, e.g. like here:

https://github.com/SerenityOS/serenity/blob/08c1effc04c08735621394b571da508fce91bd7b/Ports/python3/package.sh#L7-L18

Reply:
(this is not consistent across all ports yet, in case you copied another port script as the baseline)

Reply:
I modified a lot of settings, so it'd make sense if I defaulted to using double quotes because I forgot that this is literally a Bash script :D